benchmarks/automated/README.md

```diff
@@ -145,34 +142,44 @@ The final benchmark report will be available in markdown format as `report.md` i
 ## Features of the automation:
 1. To save time by *not* creating new instances for every benchmark run for local testing, use the '--do-not-terminate' flag. This will automatically create a file called 'instances.yaml' and write instance-related data into the file so that it may be re-used next time.
 ```
-python test/benchmark/run_benchmark.py --do-not-terminate
+python benchmarks/automated/run_benchmark.py --do-not-terminate
 ```
 
 2. To re-use an instance already recorded in `instances.yaml`, use the '--use-instances' flag:
 ```
-python test/benchmark/run_benchmark.py --use-instances <full_path_to>/instances.yaml --do-no-terminate
+python benchmarks/automated/run_benchmark.py --use-instances <full_path_to>/instances.yaml --do-no-terminate
 ```
 `Note: Use --do-not-termninate flag to keep re-using the instances, else, it will be terminated`.
 
 3. To run a test containing a specific string, use the `--run-only` flag. Note that the argument is 'string matched' i.e. if the test-name contains the supplied argument as a substring, the test will run. 
 ```
 # To run mnist test
-python test/benchmark/run_benchmark.py --run-only mnist
+python benchmarks/automated/run_benchmark.py --run-only mnist
 
 # To run fastrcnn test
-python test/benchmark/run_benchmark.py --run-only fastrcnn
+python benchmarks/automated/run_benchmark.py --run-only fastrcnn
 
 # To run bert_neuron and bert
-python test/benchmark/run_benchmark.py --run-only bert
+python benchmarks/automated/run_benchmark.py --run-only bert
 
 # To run vgg11 test
-python test/benchmark/run_benchmark.py --run-only vgg11
+python benchmarks/automated/run_benchmark.py --run-only vgg11
 
 # To run vgg16 test
-python test/benchmark/run_benchmark.py --run-only vgg16
+python benchmarks/automated/run_benchmark.py --run-only vgg16
 ```
 
 4. You can benchmark a specifc branch of the torchserve github repo by specifying the flag `--use-torchserve-branch` e.g., 
 ```
-python test/benchmark/run_benchmark.py --use-torchserve-branch issue_1115
+python benchmarks/automated/run_benchmark.py --use-torchserve-branch issue_1115
+```
+
+5. Once the docker image is built, you may choose to not have it re-built by passing the argument `--skip-docker-build` e.g.,
+```
+python benchmarks/automated/run_benchmark.py --skip-docker-build
+```
+
+6. If you do not wish to benchmark on different instance types specified in the model config `*.yaml`, you may pass the argument `--local-execution`. In this case, the instance types specified in the model config `*.yaml` file are *ignored*. Also, in this case, all the model benchmarks will be performed *sequentially* in order to 
+```
+python benchmarks/automated/run_benchmarks.py --local-execution
```

Comment:
Please add instruction about how to write the configuration files.
- docker.yaml
- modelXXX.yaml
- config.yaml

Reply:
Added details and tables describing the options for all, thanks